docs/extension/README.md

```diff
@@ -0,0 +1,26 @@
+# Extending Promgregator
+
+Promregator supports extending it. Starting with 0.10.0, Promregator not just provides runnable artifacts like the `promregator.jar` file or the Docker image, but also `promregator-library` (available through Maven repositories) is delivered. You may include this library as a dependency to your own project, for example by using the following dependency declaration in your `pom.xml`:
+
+```xml
+<dependency>
```

Comment:
@eaglerainbow - The pom is setup to provide a classifier.  This configuration probably won't work. With classifiers I have usually seen the dependency defined as such:
```
<dependency>
        <groupId>com.github.promregator</groupId>
	<artifactId>promregator</artifactId>
        <classifier>library</classifier>
	<version>0.10.0</version> <!-- or any other higher version number -->
</dependency>
```
The build will produce dependencies:

- promregator-0.10.0-SNAPSHOT.jar (The executable boot jar)
- promregator-0.10.0-SNAPSHOT-library.jar (The library jar)
- promregator-0.10.0-SNAPSHOT-sources.jar (The source code jar)
and so on.

I think your configuration noted in the PR above would expect to find "promregator-library-0.10.0-SNAPSHOT.jar" in the m2 repo. Just for clarity, my local directory would be something like:
`/development/.m2/repository/com/github/promregator/promregator-library/0.10.0-SNAPSHOT/promregator-library-0.10.0-SNAPSHOT.jar`

Reply:
You are right. I have pushed 78a67e3d78b8cc3b5cad0b9693d90655de5e53d9 which should make use of the classifier approach properly. Please check from your side!